lLyrics/LocalParser.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# Parses lyrics from a .lyric or .lrc file or from the tags of the song
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import urllib
+import os.path as path
```

Comment:
FWIW there's a modern pathlib out there, but otherwise it's not really much different.